### PR TITLE
[geometry] Test [NewObject] for DOMMatrix immutable methods

### PR DESCRIPTION
**Pull request by zcorpan**
Follows https://github.com/w3c/fxtf-drafts/pull/142

### CODE REVIEW

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/230363618)

# Firefox (nightly channel) #
Testing web-platform-tests at revision 07877dd955cee5e3f8973b888b1c18e133440ab8
Using browser at version BuildID 20170508100218; SourceStamp 1fda52a1f3b81cf1a821155998dca637bb64e3d9
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5862/css/geometry-1/DOMMatrix-newobject.html">/css/geometry-1/DOMMatrix-newobject.html</a></summary>


|               Subtest                | Results |                            Messages                           |
|--------------------------------------|---------|---------------------------------------------------------------|
|                                      | OK      |                                                               |
| `DOMMatrix translate`                | FAIL    | `Not enough arguments to DOMMatrixReadOnly.translate.`        |
| `DOMMatrix scale`                    | FAIL    | `Not enough arguments to DOMMatrixReadOnly.scale.`            |
| `DOMMatrix scale3d`                  | FAIL    | `Not enough arguments to DOMMatrixReadOnly.scale3d.`          |
| `DOMMatrix rotate`                   | FAIL    | `Not enough arguments to DOMMatrixReadOnly.rotate.`           |
| `DOMMatrix rotateFromVector`         | FAIL    | `Not enough arguments to DOMMatrixReadOnly.rotateFromVector.` |
| `DOMMatrix rotateAxisAngle`          | FAIL    | `Not enough arguments to DOMMatrixReadOnly.rotateAxisAngle.`  |
| `DOMMatrix skewX`                    | FAIL    | `Not enough arguments to DOMMatrixReadOnly.skewX.`            |
| `DOMMatrix skewY`                    | FAIL    | `Not enough arguments to DOMMatrixReadOnly.skewY.`            |
| `DOMMatrix multiply`                 | FAIL    | `Not enough arguments to DOMMatrixReadOnly.multiply.`         |
| `DOMMatrix flipX`                    | PASS    |                                                               |
| `DOMMatrix flipY`                    | PASS    |                                                               |
| `DOMMatrix inverse`                  | PASS    |                                                               |
| `DOMMatrix transformPoint`           | PASS    |                                                               |
| `DOMMatrix toFloat32Array`           | PASS    |                                                               |
| `DOMMatrix toFloat64Array`           | PASS    |                                                               |
| `DOMMatrixReadOnly translate`        | FAIL    | `Illegal constructor.`                                        |
| `DOMMatrixReadOnly scale`            | FAIL    | `Illegal constructor.`                                        |
| `DOMMatrixReadOnly scale3d`          | FAIL    | `Illegal constructor.`                                        |
| `DOMMatrixReadOnly rotate`           | FAIL    | `Illegal constructor.`                                        |
| `DOMMatrixReadOnly rotateFromVector` | FAIL    | `Illegal constructor.`                                        |
| `DOMMatrixReadOnly rotateAxisAngle`  | FAIL    | `Illegal constructor.`                                        |
| `DOMMatrixReadOnly skewX`            | FAIL    | `Illegal constructor.`                                        |
| `DOMMatrixReadOnly skewY`            | FAIL    | `Illegal constructor.`                                        |
| `DOMMatrixReadOnly multiply`         | FAIL    | `Illegal constructor.`                                        |
| `DOMMatrixReadOnly flipX`            | FAIL    | `Illegal constructor.`                                        |
| `DOMMatrixReadOnly flipY`            | FAIL    | `Illegal constructor.`                                        |
| `DOMMatrixReadOnly inverse`          | FAIL    | `Illegal constructor.`                                        |
| `DOMMatrixReadOnly transformPoint`   | FAIL    | `Illegal constructor.`                                        |
| `DOMMatrixReadOnly toFloat32Array`   | FAIL    | `Illegal constructor.`                                        |
| `DOMMatrixReadOnly toFloat64Array`   | FAIL    | `Illegal constructor.`                                        |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/230363626)

# Chrome (unstable channel) #
Testing web-platform-tests at revision de42ae89e57d62510b2a9c5cf8bdb0271106ab6f

File: css/geometry-1/DOMMatrix-newobject.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<title>Geometry Interfaces: DOMMatrix and DOMMatrixReadOnly [NewObject] methods</title>
+<link rel="help" href="https://drafts.fxtf.org/geometry/#DOMMatrix">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+function getReturnType(method) {
+  switch(method) {
+    case "transformPoint": return "DOMPoint"; break;
```
Comment (foolip) [inline css/geometry-1/DOMMatrix-newobject.html]: Nit: break after return is unreachable

**Review by foolip — APPROVED** [review]

File: css/geometry-1/DOMMatrix-newobject.html (hunk 1 above)
Comment (zcorpan) [inline css/geometry-1/DOMMatrix-newobject.html]: Yeah, I know... but it looks weird without the breaks imo ^_^

**Pull request merged**